### PR TITLE
Move the remaining crates to separate workspaces

### PR DESCRIPTION
**Pull request by rbehjati**
Ref #971 

# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,13 +1,6 @@
...
-members = [
-  "oak/server/rust/oak_loader",
-  "oak/server/rust/oak_runtime",
-  "runner",
-  "sdk/rust/oak",
-  "sdk/rust/oak_tests",
-  "third_party/roughenough",
-]
-exclude = ["oak_abi", "oak_utils"]
+members = ["oak/server/rust/oak_loader", "oak/server/rust/oak_runtime"]
```
Comment (tiziano88) [inline Cargo.toml:2]: It feels we should just move this file under oak/server?

Comment (rbehjati) [reply Cargo.toml:2]: I thought this change might break too many things and would better to be done in a separate PR. But I can give it a try here :)

File: scripts/run_tests [hunk 2]
```diff
@@ -15,7 +15,7 @@ export RUST_BACKTRACE=1
 
 for crate in "${ALL_CRATES[@]}"; do
     # Skip the experimental crates. They are covered in `build_experimental`.
```
Comment (conradgrobler) [inline scripts/run_tests:17]: Perhaps update the comment to explain why runner is also being skipped?

File: scripts/common [hunk 3]
```diff
@@ -17,7 +17,7 @@ readonly SERVER_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-server'
 readonly CACHE_DIR='bazel-cache'
 
 # List all workspaces, and standalone crates.
```
Comment (daviddrysdale) [inline scripts/common:19]: Are there any standalone crates left?

Comment (daviddrysdale) [reply scripts/common:19]: Edit: yes, I think so &ndash; `expect` and `roughenough`. 

- __`./Cargo.toml`:__
    - `./oak/server/rust/oak_loader/Cargo.toml`
    - `./oak/server/rust/oak_runtime/Cargo.toml`
- __`./examples/Cargo.toml`:__
    - `./examples/abitest/abitest_common/Cargo.toml`
    - `./examples/abitest/module_0/rust/Cargo.toml`
    - `./examples/abitest/module_1/rust/Cargo.toml`
    - `./examples/abitest/tests/Cargo.toml`
    - `./examples/aggregator/backend/Cargo.toml`
    - `./examples/aggregator/common/Cargo.toml`
    - `./examples/aggregator/module/rust/Cargo.toml`
    - `./examples/authentication/client/Cargo.toml`
    - `./examples/chat/module/rust/Cargo.toml`
    - `./examples/hello_world/module/rust/Cargo.toml`
    - `./examples/machine_learning/module/rust/Cargo.toml`
    - `./examples/private_set_intersection/module/rust/Cargo.toml`
    - `./examples/running_average/module/rust/Cargo.toml`
    - `./examples/translator/common/Cargo.toml`
    - `./examples/translator/module/rust/Cargo.toml`
    - `./examples/trusted_information_retrieval/client/rust/Cargo.toml`
    - `./examples/trusted_information_retrieval/module/rust/Cargo.toml`
- __`./experimental/Cargo.toml`:__
    - `./experimental/split_grpc/client/Cargo.toml`
    - `./experimental/split_grpc/proxy/Cargo.toml`
    - `./experimental/split_grpc/server/Cargo.toml`
- __`./oak_abi/Cargo.toml`__
- __`./oak_utils/Cargo.toml`__
- __`./runner/Cargo.toml`__
- __`./sdk/Cargo.toml`:__
    - `./sdk/rust/oak/Cargo.toml`
    - `./sdk/rust/oak_tests/Cargo.toml`
- `./third_party/expect/Cargo.toml`
- `./third_party/roughenough/Cargo.toml`

**Pull request merged**